tests/functional/tests.py

```diff
@@ -628,12 +628,23 @@ def test_negative_put_object_with_path_segment(client, log_output):
                           object_name,
                           io.BytesIO(b''), 0)
     except ResponseError as err:
-        pass
+        if err.code != 'XMinioInvalidObjectName':
+            raise Exception(err)
```

Comment:
It looks ugly to wrap an exception with `Exception` which is unnecessary.  I see this is used all over the code.

Reply:
Yes I kept it as it is.. will need to change comprehensively later.